src/device/pf_cmrpc.h

```diff
@@ -80,6 +80,23 @@ void pf_set_error (
    uint8_t code_1,
    uint8_t code_2);
 
+/**
+ * Insert detailed error information into the result structure (of a session).
+ * Does not overwrite existing error information.
+ *
+ * @param p_stat           Out:  The result structure.
+ * @param code             In:   The error_code.
+ * @param decode           In:   The error_decode.
+ * @param code_1           In:   The error_code_1.
+ * @param code_2           In:   The error_code_2.
+ */
+void pf_set_error_if_not_already_set (
```

Comment:
Is there a way to know when to use this function or the default pf_set_error()? If so a comment describing such rules is very helpful. Current solution is a bit of a workaround.

Reply:
Done